internal/codegen/tpl_stencil.go

```diff
@@ -289,3 +289,11 @@ func (s *TplStencil) ReadBlocks(fpath string) (map[string]string, error) {
 
 	return data, nil
 }
+
+// Debug logs the provided arguments under the DEBUG log level (within stencil CLI output).
```

Comment:
```suggestion
// Debug logs the provided arguments under the DEBUG log level (within stencil CLI output).
//
//  {{- $_ := stencil.Debug "I'm a log!" }}
```

Example would be nice since it's not clear how to discard the empty response to most people